src/actions/index.js

```diff
@@ -1,133 +1,147 @@
-import * as types from './types';
-import {
-  firebase,
-  applicationsRef,
-  firestore,
-  rsvpRef,
-} from '../config/firebase';
+import axios from "axios";
+import * as types from "./types";
+// import {
+//   firebase,
+//   applicationsRef,
+//   rsvpRef,
+// } from '../config/firebase';
 
-export const signUp = (values, callback) => (dispatch) => {
+const API_PATH = "https://slohacks-backend-api.herokuapp.com";
+
+export const signUp = (values, callback) => dispatch => {
```

Comment:
Consider doing something about the `values` parameter. It's very vague.

Either (a) remove it and pass in `email` and `password` or (b) change the name to like `credentials`